src/components/change-requests/change-requests-table/change-requests-filter/change-requests-filter.tsx

```diff
@@ -141,11 +142,22 @@ const ChangeRequestsFilter: React.FC<FilterFieldStateProps> = ({
           </Form.Group>
           <Button
             className={styles.applyButton}
+            onClick={() => update(type, impact, reason, state, implemented)}
+          >
+            Apply
+          </Button>
+          <Button
+            className={styles.clearButton}
             onClick={() => {
-              update(type, impact, reason, state, implemented);
+              setType('');
+              setImpact([]);
+              setReason('');
+              setState([]);
+              setImplemented('');
+              update('', [], '', [], '');
             }}
           >
-            Apply
+            Clear
           </Button>
```

Comment:
You can put these 2 buttons inside a `div` or `Form.Group` and then add on the class that has all the flex box properties